components/gitpod-cli/cmd/stop-workspace.go

```diff
@@ -17,24 +17,22 @@ var stopWorkspaceCmd = &cobra.Command{
 	Use:   "stop",
 	Short: "Stop current workspace",
 	Args:  cobra.ArbitraryArgs,
-	Run: func(cmd *cobra.Command, args []string) {
-		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
@iQQBot It would be the case where we would like rather to report an intent, i.e. we should report analytics before calling to the server.